<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="signupLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt, Arial;
color: #444444;</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="needanaccountLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>520</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Already a user?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>281</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 2px solid #D2D2D2;
	border-radius: 10px;
}
QLineEdit:focus
{
	border: 2px solid #D2D2D2;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>281</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 2px solid #D2D2D2;
	border-radius: 10px;
}
QLineEdit:focus
{
	border: 2px solid #D2D2D2;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt, Arial;
color: #444444;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="orLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;--------------------------- OR ---------------------------&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt, Arial;
color: #444444;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="singupPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#aa1111;
	border-radius: 10px;
	color: #FFFFFF;
	font: 16px;
}
QPushButton:hover
{
	background-color:#aa1111;
}
	</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: None;
	text-decoration: underline;
	color: #AA1111;
}</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmpasswordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt, Arial;
color: #444444;</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmpasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>281</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 2px solid #D2D2D2;
	border-radius: 10px;
}
QLineEdit:focus
{
	border: 2px solid #D2D2D2;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalidemailLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #AA1111;
}</string>
   </property>
   <property name="text">
    <string>*Invalid Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalidpasswordLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #aa1111;
}</string>
   </property>
   <property name="text">
    <string>*Invalid Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordsdonotmatchLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>366</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #aa1111;
}</string>
   </property>
   <property name="text">
    <string>*Passwords do not match</string>
   </property>
  </widget>
  <zorder>passwordsdonotmatchLabel</zorder>
  <zorder>invalidpasswordLabel</zorder>
  <zorder>invalidemailLabel</zorder>
  <zorder>signupLabel</zorder>
  <zorder>needanaccountLabel</zorder>
  <zorder>emailLabel</zorder>
  <zorder>orLabel</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>singupPushButton</zorder>
  <zorder>loginPushButton</zorder>
  <zorder>confirmpasswordLabel</zorder>
  <zorder>emailLineEdit</zorder>
  <zorder>passwordLineEdit</zorder>
  <zorder>confirmpasswordLineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
